allure-report-face/src/variables.css

```diff
@@ -1,10 +1,10 @@
 $brand-blue: #0044bb;
 $brand-red: #ff0000;
 
-$color-passed: #3bbd25;
-$color-failed: #ff0000;
-$color-broken: #ffa924;
-$color-canceled: #999999;
+$color-passed: #97cc64;
+$color-failed: #fd5a3e;
+$color-broken: #ffd963;
```

Comment:
Nope 
![screenshot from 2016-05-20 12-10-21](https://cloud.githubusercontent.com/assets/812240/15424598/dcf4c8b8-1e83-11e6-8610-658555df5fde.png)

Let's check this out. Current version is way more contrast